DigitalLearningSolutions.Web.Tests/Models/Enums/DlsSubApplicationTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Models.Enums
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using FluentAssertions;
+    using FluentAssertions.Execution;
+    using NUnit.Framework;
+
+    public class DlsSubApplicationTests

```

Comment:
What benefit do these tests give us?

Reply:
These are `guards` against changes to magic strings. I think it helps to re-inforce the idea that changing these values will have an impact. And also guard against "unintentional" changes.

Reply:
Aye alright, good shout.